spec/controllers/brapi/v1/studies_controller_spec.rb

```diff
@@ -0,0 +1,167 @@
+require 'rails_helper'
+require 'pp'
+
+RSpec.describe Brapi::V1::StudiesController do
```

Comment:
Sure, it should be possible. Can you share the code, e.g. push it to a temporary branch in your repo?